cloudferrylib/os/identity/keystone.py

```diff
@@ -480,9 +480,13 @@ def _render_template(self, name_file, args):
     def check_rabbitmq(self):
         credentials = pika.PlainCredentials(self.config.rabbit.user,
                                             self.config.rabbit.password)
-        for host in self.config.rabbit.hosts.split(","):
-            pika.BlockingConnection(pika.ConnectionParameters(
-                host=host.strip(), credentials=credentials))
+
+        for host_with_port in self.config.rabbit.hosts.split(","):
+            host, port = host_with_port.split(':')
```

Comment:
I guess yes, they will. At least all OpenStack services define RabbitMQ hosts in this manner (with `host:port` notation):
http://docs.openstack.org/high-availability-guide/content/_configure_openstack_services_to_use_rabbitmq.html